2024-01-16 21:27:07,072 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-01-16 21:27:07,076 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Configure stats pid to 5264
2024-01-16 21:27:07,079 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Loading settings from /home/filipeckijan/.config/wandb/settings
2024-01-16 21:27:07,081 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Loading settings from /mnt/c/Users/dziew/Documents/studia/ium/ium-projekt/models/DBSCAN/wandb/settings
2024-01-16 21:27:07,083 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-01-16 21:27:07,084 INFO    MainThread:5264 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-01-16 21:27:07,085 INFO    MainThread:5264 [wandb_init.py:_log_setup():526] Logging user logs to /mnt/c/Users/dziew/Documents/studia/ium/ium-projekt/models/DBSCAN/wandb/run-20240116_212706-ycfipzco/logs/debug.log
2024-01-16 21:27:07,087 INFO    MainThread:5264 [wandb_init.py:_log_setup():527] Logging internal logs to /mnt/c/Users/dziew/Documents/studia/ium/ium-projekt/models/DBSCAN/wandb/run-20240116_212706-ycfipzco/logs/debug-internal.log
2024-01-16 21:27:07,088 INFO    MainThread:5264 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fecadb41870>
2024-01-16 21:27:07,089 INFO    MainThread:5264 [wandb_init.py:init():566] calling init triggers
2024-01-16 21:27:07,093 INFO    MainThread:5264 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-01-16 21:27:07,099 INFO    MainThread:5264 [wandb_init.py:init():591] re-initializing run, found existing run on stack: 9hue3ar8
2024-01-16 21:27:07,104 INFO    MainThread:5264 [wandb_run.py:_finish():1962] finishing run ium-team/ium-projekt/9hue3ar8
2024-01-16 21:27:07,106 INFO    MainThread:5264 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-01-16 21:27:07,109 INFO    MainThread:5264 [wandb_run.py:_restore():2185] restore
2024-01-16 21:27:07,110 INFO    MainThread:5264 [wandb_run.py:_restore():2191] restore done
2024-01-16 21:27:12,395 INFO    MainThread:5264 [wandb_run.py:_footer_history_summary_info():3857] rendering history
2024-01-16 21:27:12,396 INFO    MainThread:5264 [wandb_run.py:_footer_history_summary_info():3889] rendering summary
2024-01-16 21:27:12,403 INFO    MainThread:5264 [wandb_run.py:_footer_sync_info():3816] logging synced files
2024-01-16 21:27:12,450 INFO    MainThread:5264 [wandb_init.py:init():616] starting backend
2024-01-16 21:27:12,451 INFO    MainThread:5264 [wandb_init.py:init():620] setting up manager
2024-01-16 21:27:12,475 INFO    MainThread:5264 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-01-16 21:27:12,476 INFO    MainThread:5264 [wandb_init.py:init():628] backend started and connected
2024-01-16 21:27:12,507 INFO    MainThread:5264 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-01-16 21:27:12,508 INFO    MainThread:5264 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-01-16 21:27:12,508 INFO    MainThread:5264 [wandb_init.py:init():720] updated telemetry
2024-01-16 21:27:12,628 INFO    MainThread:5264 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-01-16 21:27:14,034 INFO    MainThread:5264 [wandb_run.py:_on_init():2254] communicating current version
2024-01-16 21:27:14,335 INFO    MainThread:5264 [wandb_run.py:_on_init():2263] got version response 
2024-01-16 21:27:14,336 INFO    MainThread:5264 [wandb_init.py:init():804] starting run threads in backend
2024-01-16 21:27:14,538 INFO    MainThread:5264 [wandb_run.py:_console_start():2233] atexit reg
2024-01-16 21:27:14,539 INFO    MainThread:5264 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-01-16 21:27:14,539 INFO    MainThread:5264 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-01-16 21:27:14,540 INFO    MainThread:5264 [wandb_run.py:_redirect():2178] Redirects installed.
2024-01-16 21:27:14,541 INFO    MainThread:5264 [wandb_init.py:init():847] run started, returning control to user process
2024-01-16 21:27:14,548 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:27:14,548 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:27:14,559 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:27:15,279 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:27:15,280 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:27:15,301 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:27:15,303 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:27:15,304 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:27:15,321 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-16 21:27:15,328 INFO    MainThread:5264 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-16 21:27:15,335 INFO    MainThread:5264 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-16 21:27:15,356 INFO    MainThread:5264 [wandb_init.py:_resume_backend():442] resuming backend
